.changes/breaking/3002.md

```diff
@@ -0,0 +1 @@
+Update `fuel-vm` to 0.61.1
```

Comment:
please add further details of the breaking change for consumers of `fuel-core`, i.e changes to the gql schema etc

Reply:
Good point, added a comment about the `SubId` graphql change.